src/extension.ts

```diff
@@ -391,6 +436,7 @@ function logNotification(message: string, ...items: string[]) {
 function onConfigurationChange() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this function to reduce its Cognitive Complexity from 23 to the 15 allowed. ([typescript:S3776](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3776))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=18&issues=AWMmrlV1mtKPsDCoJ7QW&open=AWMmrlV1mtKPsDCoJ7QW)

[ISSUE_KEY]: AWMmrlV1mtKPsDCoJ7QW